File: src/codes/Users_Manual/gain/plane_0.py
~from pyautd3 import Intensity, Phase, Plane, PlaneOption
~nx = 1.0
~ny = 0.0
~nz = 0.0
Plane(
    direction=[nx, ny, nz],
    option=PlaneOption(
        intensity=Intensity.MAX,
        phase_offset=Phase.ZERO,
    ),
)

File: src/codes/Users_Manual/gain/plane_0.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Gain;
~var nx = 0.0f;
~var ny = 0.0f;
~var nz = 1.0f;
new Plane(
    dir: new Vector3(nx, ny, nz),
    option: new PlaneOption
    {
        Intensity = Intensity.Max,
        PhaseOffset = Phase.Zero
    }
);